lib/cooked_post_processor.rb

```diff
@@ -700,10 +705,7 @@ def html
 
   def post_process_images
     extract_images.each do |img|
-      unless add_image_placeholder!(img)
-        limit_size!(img)
-        convert_to_link!(img)
-      end
+      convert_to_link!(img) unless add_image_placeholder!(img)
```

Comment:
Does this mean we now onebox every image in a post, regardless of size?

Reply:
There is another [check (line 325)](https://github.com/discourse/discourse/pull/9224/files#diff-9cbdf282d05bfa042991776a62127eb9L325) which returns early if the image does not exceed maximum dimensions.